### PR TITLE
[zh-Hans] Upgrade to upstream main branch.

### PR DESCRIPTION
**Pull request by zdpcdt**

### CODE REVIEW

**Comment by zdpcdt** [comment]
close #259

**Review by Arhell — COMMENTED** [review]
@rabbitism

**Comment by rabbitism** [comment]
> @rabbitism 
Will review tomorrow.

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/basics/user-interface/controls/index.md [hunk 1]
```diff
@@ -28,6 +28,70 @@
...
+
+1. 尝试使用现有属性自定义控件
+2. 使用 _Avalonia UI_ 强大的样式选择器创建新样式，以修改现有模板中所需的内容
+3. 作为最后手段，重新模板化
+   :::
+
+#### 完全可定制
+
+_Avalonia UI_ 中的大多数模板化控件都是完全可定制的。这意味着可以完全替换控件的模板并更改其外观。`Button` 控件是一个很好的例子，但 _Avalonia UI_ 中的所有模板化控件都试图默认情况下是完全可定制的。对于完全可定制的模板化控件，应用程序几乎可以完全自由地样式或更改 UI 中所见的一切。
+The majority of templated controls in _Avalonia UI_ are fully customizable. This means it is possible to completely replace the control’s template and change its appearance. The `Button` control is a good example but all templated controls in _Avalonia UI_ try to be fully customizable by default. With a fully customizable templated control the app has near total ability to style or change everything you see drawn in the UI.
```
Comment (rabbitism) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/basics/user-interface/controls/index.md:85]: delete this English paragraph.

**Comment by rabbitism** [comment]
lgtm now

**Pull request merged**